src/_teradata-branding.scss

```diff
@@ -0,0 +1,629 @@
+/**
+* Colors
+*/
+
+// Customer Teradata brand palette
```

Comment:
We should import these styles instead.

Reply:
Yes, but we should update Covalent Criteria (internal use only) also